tests/search/search_export_tests.py

```diff
@@ -132,7 +199,6 @@ def test_login_via_basic_auth_rate_limited(self):
         request = RequestFactory().get(
             reverse("api_export_results"),
             HTTP_AUTHORIZATION=auth_string,
-            # In reality this would be added by django_ratelimit.
```

Comment:
Wondering if deleting this comment is intentional?

Reply:
whoops -- I had (wrongly) assumed that was an LLM artifact. Just restored it